filebeat/fileset/fileset.go

```diff
@@ -37,7 +37,6 @@ func New(
 	name string,
 	mcfg *ModuleConfig,
 	fcfg *FilesetConfig) (*Fileset, error) {
-
 	modulePath := filepath.Join(modulesPath, mcfg.Module)
```

Comment:
I updated the PR and it with from 210 files modified (485 deletions) to 198 files modified (438 deletions).